xoinvader/tests/test_collision.py

```diff
@@ -73,19 +76,23 @@ def test_collision_manager():
     gc.collect()
     assert Collider.__manager__ is None
 
+    # pylint: disable=too-few-public-methods
     class MockManager(object):
         def __call__(self):
             return self
 
         def _add(self, _collider):
             pass
 
+        def _add_collision(self, other_type, callback=None):
+            pass
+
     m = MockManager()
     Collider.__manager__ = m
     c = Collider("", [], None)
     mm = CollisionManager()
     with pytest.raises(ValueError):
-        mm._add_collision(c, "other", lambda: _)
+        mm._add_collision(c, "other", lambda: 0)
```

Comment:
Why?
`lambda: _` is a good and understandable pattern, unlike `lambda: 0`. I'm against this change.